pallets/staking/src/lib.rs

```diff
@@ -0,0 +1,261 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_election_provider_support::{
+	ElectionDataProvider, ElectionProvider, ElectionProviderBase, SortedListProvider, Supports,
+};
+use frame_support::defensive;
+use frame_support::{traits::Defensive, RuntimeDebug};
+#[cfg(feature = "std")]
+pub use pallet_staking_substrate::GenesisConfig;
+#[cfg(feature = "std")]
```

Comment:
```suggestion
#[cfg(test)]
```

Reply:
Correct me if I'm wrong, that would make it unavailable at the mocked runtime. I want to use it as a placeholder until we enable benchmarking.

Reply:
Ah right, keep forgetting the mock runtime is in a separate crate